Strings/resources.resjson/de-de/resources.resjson

```diff
@@ -0,0 +1,8 @@
+{
+  "loc.friendlyName": "Sichere Datei herunterladen",
```

Comment:
Of course you transleted all into German? ;)

Reply:
This is work from Fork repo. Will be done in next PR.

Reply:
Will be done in: https://github.com/aochmann/azure-download-multiple-secure-files/pull/2